upgrade/v110to120/upgrade.go

```diff
@@ -18,7 +18,7 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"github.com/longhorn/longhorn-manager/datastore"
-	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
```

Comment:
Why in the old upgrade path, we change to use v1beta2 instead of v1beta1 instead? I assume the new API should be used in the upgrade to 1.3.0?

Reply:
Got it, thanks.

Just curious why we don't upgrade CRs from v1beta1 to v1beta2 only just before upgrading to 1.3.0 instead of doing that at the first step before all upgrade paths?

Reply:
Discussed with @shuo-wu. It's an ordering logic consideration.
We upgrade API to v1beta2 first, and then only use the v1beta2 APIs later on.

If upgrade the CRs first, the logic will become
`upgrade longhorn version < 1.3 CRs by v1beta1` -> `upgrade API to v1beta2` -> `upgrade longhorn CRs  >= 1.3 CRs by v1beta2`.

The upgrade flow becomes complicated and duplicated.

Reply:
Thanks for the explanation